.envrc

```diff
@@ -18,6 +18,8 @@ export API_PORT=8080
 export REACT_APP_API_ADDRESS=$CLIENT_PROTOCOL://$CLIENT_DOMAIN:$API_PORT/api/v1
 export REACT_APP_GRAPHQL_ADDRESS=$CLIENT_PROTOCOL://$CLIENT_DOMAIN:$API_PORT/api/graph/query
 export REACT_APP_APP_ENV=$APP_ENV
+export LOCAL_AUTH_ENABLED=true
```

Comment:
It feels a wee bit dangerous to have the default value be `true` here. Are we sure that will get overwritten in our deployed environments? Could we ask folks to drop these in .envrc.local and change the values there?

Reply:
AFAIK we don't use `.envrc` at all in deployed environments. @jsclarridge can you confirm or deny?

Reply:
@jim Correct, the environment variable values for the deployed environments aren't pulled from `.envrc`. That being said, we should do a dev deployment on this branch before merging.

Reply:
@sarboc Do you think it's OK to leave the values here, then, or should we move them?